drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -235,7 +242,12 @@ def service_form_definition(service_type_id:, project_id:)
       service_type = Hmis::Hud::CustomServiceType.find_by(id: service_type_id)
       raise HmisErrors::ApiError, 'Service type not found' unless service_type.present?
 
-      Hmis::Form::Definition.find_definition_for_service_type(service_type, project: project)
+      definition = Hmis::Form::Definition.find_definition_for_service_type(service_type, project: project)
+
+      # Similar to record_form_definition above, we always want to return a definition if we possibly can
+      # TODO @MARTHA - this identifier could be different across environments, what is the best way to find the most basic service to return?
+      # service, hud-service-with-note, ...
+      definition || Hmis::Form::Definition.with_role(:SERVICE).where(identifier: 'service-date-only').first
```

Comment:
Use "service" identifier and add a todo for https://github.com/open-path/Green-River/issues/6763

Reply:
@gigxz using the system service form doesn't quite work nicely for custom services. I don't feel that confident about the workaround I've added here: https://github.com/greenriver/hmis-warehouse/pull/4867/files#diff-de4ce2db2cf589590502110af670f97dd57d9aae64e79a4615a3ece0644acbe3R16-R18. It would break if the user is editing a custom service and trying to change a filled-in value to nil.

Reply:
https://github.com/greenriver/hmis-warehouse/pull/4867#discussion_r1829480896